src/main/java/cubicchunks/CommonEventHandler.java

```diff
@@ -34,12 +37,19 @@
 
 	@SubscribeEvent
 	public void onWorldLoad(WorldEvent.Load evt) {
-		if (!(evt.getWorld().getWorldType() instanceof ICubicChunksWorldType) ||
-				evt.getWorld().provider.getDimension() != 0) {
+		if (!(evt.getWorld().getWorldType() instanceof ICubicWorldType)) {
 			return;
 		}
+
 		CubicChunks.LOGGER.info("Initializing world " + evt.getWorld() + " with type " + evt.getWorld().getWorldType());
 		ICubicWorld world = (ICubicWorld) evt.getWorld();
+
+		WorldType type = evt.getWorld().getWorldType();
```

Comment:
I know, but the first part of the method is the logic that determines weather or not to initialize Cubic Chunks on a world... I wanted to keep it separate from the rest of the method.

World types are not even needed to make a world cubic, and I am even thinking of adding a toggle button in the world creation menu. :)